assets/js/modules/analytics/components/settings/SettingsMain.test.js

```diff
@@ -27,9 +27,11 @@ import SettingsMain from './SettingsMain';
 
 describe( 'SettingsMain', () => {
 	const initialSettings = {
+		/* eslint-disable sitekit/camelcase-acronyms */
 		accountID: fixtures.accountsPropertiesProfiles.profiles[ 0 ].accountId,
 		propertyID: fixtures.accountsPropertiesProfiles.profiles[ 0 ].webPropertyId,
 		internalWebPropertyID: fixtures.accountsPropertiesProfiles.profiles[ 0 ].internalWebPropertyId,
+		/* eslint-enable sitekit/camelcase-acronyms */
```

Comment:
Not a huge deal, but would it be technically possible to only disable the rule partially? For example, in each of these three lines, having one violation is acceptable (for the latter property reference), but having two violations in one line would mean that we also violated the rule where we shouldn't have (in the property name that is set).

Reply:
Sadly, there's no way to do that as I understand it. 😞